pages/publishing/publishing/amplitude-data.mdx

```diff
@@ -0,0 +1,78 @@
+import {Callout} from 'nextra/components'
+
+# Amplitude Data
+
+*Publishing your tracking plan to Amplitude Data*
+
+![Amplitude Logo](/images/publishing_Amplitude Data.svg)
+
+## Introduction
+
+
+The integration to [Amplitude Data](https://help.amplitude.com/hc/en-us/categories/17292443498779-Amplitude-Data) allows for your Tracking Plan to be published into Amplitude Data whenever a branch has been merged. You can also manually trigger a publish by clicking the "Publish to Amplitude Data" button on the integration screen or download a CSV file on the Amplitude Data format, which you can manually import into Amplitude Data. The events to be published can be [filtered by Sources, Destinations and Tags](/publishing/publishing/overview#filtering-events-for-publishing).
+
+## Configure the Amplitude Data integration
+
+To enable the direct integration with Amplitude Data, you need to configure it with the following parameters from Amplitude. They can be found from the project settings in Amplitude. From the left sidebar in your Amplitude workspace click: Settings > Projects > [Your Project Name]. The URL should look like this: `https://analytics.amplitude.com/{org-name}/settings/projects/{project-id}/general`.
+
+- **Amplitude API Key**
+- **Amplitude Secret Key**
```

Comment:
Let me confirm with Amplitude if each project still needs to be configured for this to work: https://avosh.slack.com/archives/C01G4LXUDGE/p1689943090014029

Reply:
We should update the docs accordingly: "customers should reach out to their CSM or customer support. We have a note about this in our docs [here](https://www.docs.developers.amplitude.com/analytics/apis/taxonomy-api/)"